server/meshmodel/components/kubernetes/meshmodel_metadata.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If this is statically registered, hopefully, we will pull from the capabilities registry as the source of truth...

Reply:
We'll need this item wrapped up.